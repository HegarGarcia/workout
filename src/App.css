@import url("https://fonts.googleapis.com/css2?family=Fugaz+One&family=Roboto+Mono:wght@500&family=Roboto:wght@300;400;500&display=swap");

* {
  box-sizing: border-box;
}

html,
body,
div#root {
  width: 100%;
  height: 100%;

  font-family: "Roboto", sans-serif;
}
