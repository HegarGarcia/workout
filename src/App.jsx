import { blue, green, grey, red } from '@material-ui/core/colors';
import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
import 'normalize.css';
import React from 'react';
import ContextProvider from './context/provider';
import Router from './router/Router';
import GlobalStyle from './styles/GlobalStyle';

const theme = createMuiTheme({
  palette: {
    type: 'dark',
    primary: {
      main: blue.A400
    },
    secondary: {
      main: green.A400,
      contrastText: '#fff'
    },
    warning: {
      main: red.A200,
      contrastText: '#fff'
    },
    background: {
      default: grey[900],
      paper: grey[900]
    }
  }
});

export default function App() {
  return (
    <>
      <GlobalStyle />
      <ContextProvider>
        <ThemeProvider theme={theme}>
          <Router />
        </ThemeProvider>
      </ContextProvider>
    </>
  );
}
