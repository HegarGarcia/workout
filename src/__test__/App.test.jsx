import React from 'react';
import { render, act } from '@testing-library/react';
import App from '../App';

describe('<App /> spec', () => {
  it('Should not regress', async () => {
    let container;

    await act(async () => {
      container = await render(<App />);
    });

    expect(container.container).toMatchSnapshot();
  });
});
