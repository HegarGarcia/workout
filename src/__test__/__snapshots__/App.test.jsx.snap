// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> spec Should not regress 1`] = `
<div>
  <div
    class="sc-AxheI bPuNAc"
  >
    <main
      class="sc-AxhUy efgNkU"
    >
      <section
        class="sc-AxjAm sc-AxirZ hcRbbY"
      >
        <div
          class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
          role="progressbar"
          style="width: 40px; height: 40px;"
        >
          <svg
            class="MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </div>
      </section>
    </main>
  </div>
</div>
`;
